.release-it.yaml

```diff
@@ -1,17 +0,0 @@
-git:
```

Comment:
Any specific reason for moving this from yaml to json? Could we keep it in yaml for less syntax overhead?

Reply:
I moved it to JSON because it better matched the documentation of release-it, but I can change it back if you prefer.

Reply:
In that case, JSON is fine by me. Thanks for the update.